app/packages/core/src/components/Actions/Tagger.tsx

```diff
@@ -373,6 +387,18 @@ const useTagCallback = (
         } else if (samples) {
           set(fos.refreshGroupQuery, (cur) => cur + 1);
           updateSamples(samples.map((sample) => [sample._id, sample]));
+
+          if (isImaVidLookerActive) {
+            // assuming we're working with only one sample
+            (
+              lookerRef?.current as unknown as ImaVidLooker
+            )?.frameStoreController.store?.updateSample(
+              samples[0]._id,
+              samples[0]
+            );
+
+            lookerRef?.current?.updateSample(samples[0]);
+          }
```

Comment:
Maybe a `looker.updateSamples` method could be added? Only one sample can updated currently, but this would future proof selection/tagging changes and avoid accessing internal Looker data structures